### PR TITLE
10.0 website event sale multi company

### PR DESCRIPTION
**Pull request by iledarn**

### CODE REVIEW

File: website_multi_company_event_sale/README.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/lgpl
+   :alt: License: LGPL-3
+
+===============
+ Real Multi Website (Online Event's Tickets extension)
+===============
```
Comment (yelizariev) [inline website_multi_company_event_sale/README.rst]: fix format

File: website_multi_company_event_sale/README.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Convigure Events' Tickets per website
+
+{Then add more detailed description, technical specifications, any other information that could be interested for other developers. Don't forget that Usage instructions is a separated and has to be located in doc/index.rst file.}
```
Comment (yelizariev) [inline website_multi_company_event_sale/README.rst]: add more detailed description,

**Comment by yelizariev** [comment]
You can ignore this lint warning

``portal_event_tickets/models/event_mail.py:59: [W8104(api-one-deprecated), EventMailRegistration._compute_scheduled_date] api.one deprecated`` 
-- it doesn't affect lint status

File: website_multi_company_event_sale/views/event_views.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (yelizariev) [inline website_multi_company_event_sale/views/event_views.xml]: Add copyrights

**Comment by iledarn** [comment]
@yelizariev , done

File: website_multi_company_event_sale/security/event_security.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (yelizariev) [inline website_multi_company_event_sale/security/event_security.xml]: add copyrights

**Pull request merged**